mdast_util_to_markdown/src/state.rs

```diff
@@ -27,9 +27,9 @@ use regex::{Captures, Regex, RegexBuilder};
 
 #[derive(Debug)]
 enum Join {
```

Comment:
I've created these join variants name out of my understating for how _join_ works, in JS the returned type are a bit confusing tbh and I would rather have these to describe the type of a _Join_  in Rust